### PR TITLE
TST add corner cases in test_constructors

### PR DESCRIPTION
**Pull request by r-toroxel**
add two more corner cases to frame/test_constructors.

### CODE REVIEW

File: pandas/tests/frame/test_constructors.py [hunk 1]
```diff
@@ -1168,6 +1168,12 @@ def test_constructor_list_of_odicts(self):
             OrderedDict([["b", 3], ["c", 4], ["d", 6]]),
         ]
 
+        # single row dataframe
```
Comment (WillAyd) [inline pandas/tests/frame/test_constructors.py:1171]: Can you create this as a separate test?

**Comment by jreback** [comment]
you have a lint error: https://github.com/pandas-dev/pandas/pull/35216/checks?check_run_id=858699930

you can use the precommit hooks to automatically fix this.

**Comment by r-toroxel** [comment]
convenient. installed.

**Comment by jreback** [comment]
thanks @r-toroxel

**Pull request merged**